helm_deploy/hmpps-resettlement-passport-api/files/stubs/mappings/interventions-service-api-getCRSAppointments-A8731DY-200.json

```diff
@@ -1,7 +1,7 @@
 {
   "request": {
     "method": "GET",
-    "urlPath": "/interventions-service-api/referral/709f9681-0dca-4f95-9d3b-59bd9322c6f4/delivery-session-appointments"
+    "urlPath": "/appointments-location/U328968"
```

Comment:
it's missing the mapping for URN U338861, NomisID B1234ZX 

`helm_deploy/hmpps-resettlement-passport-api/files/stubs/mappings/interventions-service-api-getCRSAppointments-B1234ZX-200.json`